labgrid/driver/sshdriver.py

```diff
@@ -389,12 +391,28 @@ def get_status(self):
         """The SSHDriver is always connected, return 1"""
         return 1
 
+    def _ssh_prefix_append_s(self):
+        version = subprocess.run("ssh -V", capture_output=True, shell=True, text=True)
+        version = re.match(r'^OpenSSH_(\d+)\.(\d+)(?:\.\d+){0,2}\w*', version.stderr)
+        major = int(version[1])
+        minor = int(version[2])
+
+        if major == 8 and minor >= 6:
+            return True
+        elif major >=9:
+            return False
+        else:
+            raise Exception(f'OpenSSH version {major}.{minor} is not supported. It must be at least version 8.6')
```

Comment:
```suggestion
        # OpenSSH >= 8.6 supports explicitly using the SFTP protocol via -s
        if major == 8 and minor >= 6:
            return True
        # OpenSSH >= 9.0 default to the STFTP protocol
        if major >= 9:
            return False
        raise Exception(f"OpenSSH version {major}.{minor} does not support explicit SFTP mode")
```
Replaced an unnecessary `elif` with `if` and dropped an `else`. Added comments to the cases and simplified the exception.

Reply:
OK for me.